tests/test_watermark_no_torch_compile_1615.py

```
@@ -0,0 +1,188 @@
+    seen: list[bool] = []
+    errors: list[BaseException] = []
+
+    def first():
+        try:
+            with watermark._eager_audioseal():
+                first_in.set()
+                assert second_in.wait(timeout=10), "second scope never opened"
+            first_out.set()                      # closes while `second` is open
+        except BaseException as exc:  # noqa: BLE001 — surfaced via the assert below
+            errors.append(exc)
+            first_in.set(); first_out.set()
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)